db/seeds/prisons/LCI-leicester.yml

```diff
@@ -33,5 +33,4 @@ slots:
 unbookable:
 - 2014-12-25
 - 2014-12-26
-- 2014-12-31
```

Comment:
Indeed! Though I think those things had been caught in the old app between the time when we initially imported the data and the time when you added YAML checking over there.